### PR TITLE
link data_manager to VaultFacade; implement DataManager::handle_get

### PR DESCRIPTION
**Pull request by maqi**

### CODE REVIEW

**Comment by dirvine** [comment]
Sorry Qi this has now conflicted with my rush through to work with nightly builds again (which you can now use) can you please
```curl -s https://static.rust-lang.org/rustup.sh | sudo sh```
and merge / push

**Comment by maqi** [comment]
closing this PR as it has been resolved already in other way

**Comment by maqi** [comment]
reopen as new commits need to be merged

File: src/chunk_store.rs [hunk 1]
```diff
@@ -12,7 +12,9 @@
     See the Licences for the specific language governing permissions and limitations relating to
     use of the MaidSafe
     Software.                                                                 */
-  
+
+#![allow(dead_code)]
```
Comment (dirvine) [inline src/chunk_store.rs]: Should thi s have a /// FIXME associated

File: src/vault.rs [hunk 2]
```diff
@@ -13,54 +13,54 @@
...
-//   }
-// }
+#![allow(unused_variables)]
+
+extern crate Routing;
+
+#[path="data_manager/data_manager.rs"]
+mod data_manager;
+
+use self::Routing::Authority;
```
Comment (dirvine) [inline src/vault.rs]: Nice

**Pull request merged**